readthedocs/templates/profiles/private/security_log.html

```diff
@@ -0,0 +1,80 @@
+{% extends "profiles/base_profile_edit.html" %}
+
+{% load i18n %}
+{% load pagination_tags %}
+
+{% block title %}{% trans "Security Log" %}{% endblock %}
+
+{% block profile-admin-security-log %}active{% endblock %}
+
+{% block edit_content_header %} {% trans "Security Log" %} {% endblock %}
+
+{% block edit_content %}
+
+<p>
+  {% blocktrans trimmed with days_limit as days_limit %}
+  Showing logs from the last {{ days_limit }} days.
+  {% endblocktrans %}
+</p>
+
+{% autopaginate object_list 15 %}
+
+<div class="module">
+  <form method="get">
+    <button type="submit" name="download" value="true">{% trans "Download all data" %}</button>
+  </form>
+
+  <div class="module-list">
+    <div class="module-list-wrapper">
+      <ul>
+        {% for log in object_list %}
+        <li class="module-item">
+          {% if log.action == 'authentication' %}
+            {% blocktrans trimmed %}
+            <a href="?action=authentication">Authenticated</a>
+            {% endblocktrans %}
+          {% elif log.action == 'authentication-failure' %}
```

Comment:
I feel like we want some kind of constant for these. It seems error-prone to use the strings directly.